doc/sphinx/addendum/type-classes.rst

```diff
@@ -289,8 +302,27 @@ declares each projection as an instance.
 .. warn:: Ignored instance declaration for “@ident”: “@term” is not a class
 
    Using this ``:>`` syntax with a right-hand-side that is not itself a Class
-   has no effect (apart from emitting this warning). In particular, is does not
-   declare a coercion.
+   has no effect (apart from emitting this warning).
+
+.. warn:: 'Class >' currently does nothing, a coercion will be introduced in future versions.
+
+   :g:`Class >` will behave as :g:`Record >` in future versions,
+   declaring the construcotr a coercion. Remove the :g:`>` to avoid
+   this warning.
+
+.. warn:: 'Class >' is not supported for singleton typeclasses.
+
+   :g:`Record >` make the constructor a coercion, but there is no
+   constructor for singleton classes. Remove the :g:`>` to avoid this
+   warning.
```

Comment:
Is it possible to combining this message and the previous message into one?  They're both about `Class >` and coercions, but the distinction doesn't seem interesting.  Unless, perhaps, the idea is that this one will not change in the future.

Reply:
Indeed, this will not change (singleton classes have no constructor to turn into a coercion, so the syntax simply doesn't make any sense).